src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -52,15 +52,25 @@
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " 1 "
             + PREFIX_PHONE + "91234567 "
-            + PREFIX_EMAIL + "johndoe@example.com";
+            + PREFIX_EMAIL + "johndoe@example.com\n"
+            + "OR "
+            + "Edit the specified tag in all contacts containing this tag with a new specified tag "
+            + "Parameters: [old/TAG] [new/TAG] "
```

Comment:
Should import constant prefixes from CliSyntax class here for consistency

Reply:
Done.